pkg/resource/logging-config/alloy/logging.alloy.template

```diff
@@ -89,7 +89,25 @@ discovery.relabel "kubernetes_pods" {
 		target_label  = "container"
 	}
 
-{{- if not .SupportPodLogs }}
+	{{- if .SupportPodLogs }}
+	rule {
+		source_labels = ["giantswarm_observability_tenant"]
+		target_label  = "__tenant_id__"
+	}
+
+	// We drop the label to not have it ingested in loki
+	rule {
+		regex = "giantswarm_observability_tenant"
+		action = "labeldrop"
+	}
+
+	// We drop any log lines that do not contain a tenant_id label
+	rule {
+		source_labels = ["__meta_kubernetes_pod_container_name", "__tenant_id__"]
+		regex = "(.+);"
+		action = "drop"
+	}
```

Comment:
Do we really need that ? I would assume that those would be rejected. This would make those silently fails in this case.

Reply:
An error metric paging is not silent :)

Reply:
This will create a lot of warn logs in alloy as well as a lot of 401 in the loki gateway and nginx which is not really good imo. I prefer that we drop to avoid those

Reply:
ok why not